compiler/test/formatter_outputs/variants.gr

```diff
@@ -0,0 +1,12 @@
+enum Animals {
+  VeryVeryLong,
+  VeryVeryLongVeryVeryLong,
+  VeryVeryLongVeryVeryLongVeryVeryLong,
+}
+
+enum SmallAnimals { Abc, Def, }
+
+enum CabbageColor { Red, Green, }
+enum Veggie { Squash, Cabbage(CabbageColor), Broccoli, }
```

Comment:
I'm still in the no trailing comma if they all fit on the same line camp 😛

Reply:
I think I am too.  I took this short cut of always adding a trailing comma because of adding trailing comments (easier to always add the comma as I can then always safely add a comment after) but I think the printer has support for that.

Reply:
Fixed.